tests/table/test_aggregate.py

```diff
@@ -0,0 +1,54 @@
+from piccolo.query import Avg, Max, Min, Sum
+
+from ..base import DBTestCase
+from ..example_app.tables import Band
+
+
+class TestAggregate(DBTestCase):
+    def test_avg(self):
+        self.insert_rows()
+
+        response = Band.select(Avg(Band.popularity)).first().run_sync()
+
+        self.assertTrue(float(response["avg"]) == 1003.3333333333334)
```

Comment:
This is great. I think it can also be useful to allow `as_alias` for aggregate functions the same way as `Column` does. What do you think?

Reply:
@aminalaee Thanks for great suggestion.